package/kernel/linux/modules/netdevices.mk

```diff
@@ -538,6 +538,56 @@ endef
 
 $(eval $(call KernelPackage,dsa))
 
+
+define KernelPackage/dsa-tag-brcm
+  SUBMENU:=$(NETWORK_DEVICES_MENU)
+  TITLE:=Broadcom DSA tagging support
+  DEPENDS:=+kmod-dsa
+  KCONFIG:= CONFIG_NET_DSA_TAG_BRCM_COMMON
```

Comment:
`CONFIG_NET_DSA_TAG_BRCM_COMMON` does not have a title in the kernel kconfig, you can not select it here. 

You should select `CONFIG_NET_DSA_TAG_BRCM`, `CONFIG_NET_DSA_TAG_BRCM_LEGACY` and `CONFIG_NET_DSA_TAG_BRCM_PREPEND` here.

You can make this package hidden too with `HIDDEN:=1`, it does not make sense to select it without `kmod-dsa-b53`.

Reply:
Thanks for your review, Hauke. As I just wrote [above](https://github.com/openwrt/openwrt/pull/11680#discussion_r1790993472), I realized I didn't selecting the `CONFIG_NET_DSA_TAG_BRCM_COMMON` option here, but including the `tag_brcm.ko` file in the `kmod-dsa-b53` package, as I am doing it now.

Closing this to avoid multiple review threads.